rest/taskana-rest-spring/pom.xml

```diff
@@ -69,6 +69,14 @@
             <groupId>org.springframework</groupId>
             <artifactId>spring-webmvc</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-autoconfigure</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>com.unboundid</groupId>
+            <artifactId>unboundid-ldapsdk</artifactId>

```

Comment:
This should be a test dependency, shouldn't it? (even though it didn't has the scope before)

Reply:
I think so. done